src/account.ts

```diff
@@ -0,0 +1,58 @@
+import { Coin, DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+import {SigningStargateClient, StargateClient, assertIsDeliverTxSuccess } from "@cosmjs/stargate";
+import { extractAddress, ArchwaySigningClient } from "./connect";
+
+// Get account info of Signing Client
+export async function getAccount(wallet: DirectSecp256k1HdWallet) {
+    const account = await extractAddress(wallet)
+    return account
+}
+
+// Get balance of client wallet
+ArchwaySigningClient.prototype.getBalance = async function() {
+  const address = await extractAddress(this.wallet);
+  const balance = await this.client.getBalance(address, "uconst");
+  return balance
+}
+
+// Send tokens to another Archway address
+ArchwaySigningClient.prototype.sendTokens = async function(archway_recipient_address: string, amount: Coin[]) {
+    const wallet = this.wallet;
+    const wallet_address = await wallet.getAccounts()
+    const testnet_client = this.client;
+    const fee = {
+        amount: [
+          {
+            denom: "uconst",
```

Comment:
shouldn't those come from a config? I meant `denom`, `amount` and `gas`

Reply:
Having issue stabilizing fees, will fix to be an argument in near PR.

Reply:
Why not change in this pr?